glsl/glsl-lib.js

```diff
@@ -504,7 +512,7 @@ vec4 sphericalDistribution( vec3 p, float n )
 
     float k = max(2.0, floor( log(n * PI * sqrt(5.0) * (1.0 - cosTheta*cosTheta))/ log(PHI+1.0)));
     float Fk = pow(PHI, k)/sqrt(5.0);
-    vec2 F = vec2( round(Fk), round(Fk * PHI) ); // k, k+1
+    vec2 F = vec2( floor(Fk + 0.5), floor(Fk * PHI + 0.5) ); // k, k+1
```

Comment:
Was this for webgl1 compatibility? We did migrate shaderpark to webgl2 a while ago.

Reply:
I see. I suspect it's possible there may be other places in the library that could have issues with webg1 besides this line. I'm happy to merge now and just fix them later if they show up.